linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_groupownership.rule

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
+prodtype: rhel6,rhel7,fedora,ol7
```

Comment:
I'm fine with dropping the prodtype here, since after the debian8 merger, it'd look like

`prodtype: debian8,ubuntu1404,ubuntu1604rhel6,rhel7,fedora,ol7`

and become way too messy.

Reply:
No problem with dropping this off if it is expected to work on all affected prodtypes.